src/models/RoversResponse.ts

```diff
@@ -0,0 +1,21 @@
+import { Camera } from "./Camera"
+import { Rover } from "./Rover"
+
+export type RoversResponse = {
+    rovers: {
+        id: number,
+        name: Rover,
+        landing_date: string,
+        launch_date: string,
+        status: string, 
+        max_sol: number,
+        max_date: string,
+        total_photos: number,
+        cameras: [
+            {
+                name: Camera,
+                full_name: string
+            }
+        ]
+    }[]
+}
```

Comment:
```suggestion
export inteface RoversResponse {
    rovers: {
        id: number,
        name: Rover,
        landing_date: string,
        launch_date: string,
        status: string, 
        max_sol: number,
        max_date: string,
        total_photos: number,
        cameras: [
            {
                name: Camera,
                full_name: string
            }
        ]
    }[]
}
```

nitpick: Conventionally in Typescript, we use interface when we can. `type` would be useful when you're defining a union / intersection types e.g.
```
type A = B | C
type D = E & F
```